Build/Chakra.Build.props

```diff
@@ -55,6 +55,16 @@
         %(PreprocessorDefinitions);
         INTL_ICU=1
       </PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(ChakraICU)'=='static'">
+        %(PreprocessorDefinitions);
+        U_STATIC_IMPLEMENTATION=1
+      </PreprocessorDefinitions>
+      <!-- Act like we are using Windows Kit ICU -->
```

Comment:
If you really want to mimic the SDK/Kit ICU version then consider perhaps adding the following:
`U_DEFAULT_SHOW_DRAFT=0; U_HIDE_DRAFT_API=1; U_HIDE_DEPRECATED_API=1; U_HIDE_OBSOLETE_API=1; U_HIDE_INTERNAL_API=1;`

Reply:
I added these to CommonDefines.h so they they don't interfere with the in-tree build of ICU. Thanks for the suggestion!